assets/src/dashboard/animations/bounce/index.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+function Bounce({ style, duration, delay, children }) {
+  return (
+    <div
+      style={{
+        animation: `bounce ${duration} ${delay} forwards`,
+        transform: 'scale(0)',
+        ...(style || {}),
+      }}
+    >
+      {children}
+    </div>
+  );
+}
+
+Bounce.propTypes = {
+  style: PropTypes.object,
+  duration: PropTypes.string,
+  delay: PropTypes.string,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+};
+
+Bounce.defaultProps = {
+  duration: '2250ms',
+  delay: '100ms',
```

Comment:
That should always be defaulted at `0` and only customizing for compound animations. If an animation has a inherent delay, that should be either a curve or a keyframes set up.